crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -484,12 +487,12 @@ impl PipelineCache {
         &mut self,
```

Comment:
This doesn't need to be `&mut self` anymore, no?

Reply:
Yes, sorry, silly omission.